pom.xml

```diff
@@ -20,7 +20,7 @@
     <parent>
         <groupId>org.apache</groupId>
         <artifactId>apache</artifactId>
-        <version>25</version>
+        <version>31</version>
```

Comment:
Hi @anirudh-hegde , thanks for contribute this. Could you explain for me what's different of the version changed?

Reply:
I thought newer versions would get better community support and also version 31 introduces new functionalities compatible with complex applications or platforms like Aiven.